create-table.md

```diff
@@ -1,22 +1,111 @@
-|  |  |
\ No newline at end of file
+## Semantics
+
+### IF NOT EXISTS
+
+Do not raise a **table already exists** error if a table of the same name in the database already exists. Note that the existence check is on the table name only, there is no guarantee that the existing table has the same columns, indexes, constraints, etc as the one that would have been created. 
+
+
+#### any_name
+
+The [name](data-definition.html#identifiers) (optionally schema-qualified) of the table to be created. Table names are required to be unique within a database.
+
+The `UPSERT` and `INSERT INTO ... ON CONFLICT` statements make use of a table called `EXCLUDED` to handle data conflicts during execution. It's therefore not recommended to use the name `EXCLUDED` for any of your tables.
+
+
+#### column_def
+
+An optional comma separated list of any [column names](data-definition.html#identifiers), their [data types](data-types.html), and any [column level constraints](data-definition.html#column-level-constraints). Column names have their own namespace within a table's definition so are required to be unique for the table but may have the same name as indexes or constraints, however this is not recommended. Any  Primary Key, Unique, and Check constraints that were originally defined at the column level are moved to the table level as part of the table's creation. Using the `SHOW CREATE TABLE` command will show these constraints at the table level.
+
+
+#### index_def
+
+An optional comma separated list of any [index definitions](data-definition.html#indexes). Index names have their own namespace within a table's definition so are required to be unique for the table but may have the same name as columns or constraints, however this is not recommended.
+
+
+#### table_constraint
+
+An optional comma separated list of any [table level constraints](data-definition.html#table-level-constraints). Constraint names have their own namespace within a table's definition so are required to be unique for the table but may have the same name as columns or indexes, however this is not recommended. 
+
+
+## Usage
+
+If the table does not have a `PRIMARY KEY` constraint defined, a mandatory column called `rowid` of type `INT` will be added and automatically populated with a unique id that will be used as the primary key. The column will be added even if the table has an `UNIQUE` constraint.
+
+To display a list of tables created in a database, use the [`SHOW TABLES`](show-tables.html) command.
+
+To display information about columns in a table, use the [`SHOW COLUMNS`](show-columns.html) command.
+
+To show the definition of a table, use the [`SHOW CREATE TABLE`]() command. 
+
+By default, tables are created in the default replication zone but can be placed into a specific replication zone. See the discussion on [Creating a Replication Zone for a Table](configure-replication-zones.html#create-a-replication-zone-for-a-table) for more information.
+
+## Examples
```

Comment:
Instead of one example containing all complexities, can you try to break this into a few separate examples?